Documentation/community/community.rst

```diff
@@ -4,25 +4,38 @@
     Please use the official rendered version released here:
     https://docs.cilium.io
 
-.. _weekly-community-meeting:
+.. _community-meeting:
 
-Weekly Community Meeting
-========================
+Community Meetings
+==================
 
-The Cilium contributors gather every Wednesday at 8am PDT, 17:00 CEST, for a
-Zoom call open to everyone. During that time, we discuss:
+The Cilium contributors gather regularly for a Zoom call open to everyone.
+During that time, we discuss:
 
 - Status of the next releases for each supported Cilium release
 - Current state of our CI: flakes being investigated and upcoming changes
 - Development items for the next release
 - Any other community-relevant topics during the open session
 
-If you want to discuss something during the next meeting's open session,
-you can add it to `the meeting's Google doc
-<https://docs.google.com/document/d/1Y_4chDk4rznD6UgXPlPvn3Dc7l-ZutGajUv1eF0VDwQ/edit#>`_.
-The Zoom link to the meeting is available in the ``#development`` Slack
-channel and in `the meeting notes
-<https://docs.google.com/document/d/1Y_4chDk4rznD6UgXPlPvn3Dc7l-ZutGajUv1eF0VDwQ/edit#>`_.
+If you want to discuss something during the next meeting's open session, you
+can add it to the meeting's Google doc. The Zoom link to the meeting is
+available in the ``#development`` Slack channel and in the meeting notes.
+
+Weekly Community Meeting
+------------------------
+
+This is a weekly meeting for all contributors.
+
+- Date: Every Wednesday at 8:00 PDT, 17:00 CEST
```

Comment:
I would probably suggest to drop CEST. Given that twice a year for about a month total, the meeting shifts depending on DST, specifying both timezones is confusing.

I'm not sure the canonical way to specify the time zone but this tracks US Pacific time which technically switches between PST and PDT depending on time of year. Maybe the "US/Pacific (Los Angeles)" nomenclature (like in Linux system settings) is clearer?

Reply:
Makes sense. Changed to the "US/Pacific" notation.

Reply:
> I'm not sure the canonical way to specify the time zone

I understand “PT” (Pacific Time) alternates between PST and PDT; but the “US/Pacific” notation seems less confusing :+1:.